hooks/useLocal.ts

```diff
@@ -219,9 +219,22 @@ export const useLocal = () => {
   const fetchConsent = async () => {
     try {
       const consent = await AsyncStorage.getItem('consent');
-      return consent === null ? true : consent === 'true';
+      return consent === 'true';
     } catch (error) {
-      return true;
+      return false;
+    }
+  };
+
+  const initConsentIfMissing = async () => {
```

Comment:
Instead of creating a separate function, you can add this in fetchConsent itself. Right now, there are two functions that return the consent value. The return statements from `initConsentIfMissing` is not being used anywhere.

```js
  const fetchConsent = async () => {
    try {
      const consent = await AsyncStorage.getItem('consent');
      if (consent === null) {
        await AsyncStorage.setItem('consent', 'true');
        return true;
      }
      return consent === 'true';
    } catch (error) {
      return false;
    }
  };
```